deeppavlov/core/data/simple_vocab.py

```diff
@@ -29,7 +29,16 @@
 
 @register('simple_vocab')
 class SimpleVocabulary(Estimator):
-    """Implements simple vocabulary."""
+    """Implements simple vocabulary.
+
+    Parameters:
+        special_tokens: tuple of tokens that shouldn't be counted.
+        max_tokens: TODO
+        min_freq: minimal count of a token (except special tokens).
+        pad_with_zeros: TODO
+        unk_token: label assigned to unknown tokens.
+        freq_drop_load: TODO
```

Comment:
Who can update this?

Reply:
May be @kudep ? He may know the best description of the params as he authored the `__init__`'s signature

Reply:
Documented, thanks to @kudep

Reply:
I transferred all my mind's about that to @acriptis, now he can description it))